engine/xml-config/src/main/java/org/datacleaner/configuration/DatastoreXmlExternalizer.java

```diff
@@ -415,6 +422,31 @@ public Element toElement(SalesforceDatastore datastore) {
     }
 
     /**
+     * Externalizes a {@link DataHubDatastore} to an XML element
+     * 
+     * @param datastore
+     * @return
+     */
+    private Element toElement(DataHubDatastore datastore) {
+        final Element ds = getDocument().createElement("datahub-datastore");
+        ds.setAttribute("name", datastore.getName());
+        if (!StringUtils.isNullOrEmpty(datastore.getDescription())) {
+            ds.setAttribute("description", datastore.getDescription());
+        }
+
+        appendElement(ds, "host", datastore.getHost());
+        appendElement(ds, "port", datastore.getPort());
+        appendElement(ds, "username", datastore.getUsername());
+        appendElement(ds, "password", encodePassword(datastore.getPassword()));
+        appendElement(ds, "tenantname", datastore.getTenantName());
+        appendElement(ds, "https", datastore.https());
```

Comment:
Not related to this PR, but the method name https() should probably be "isHttps()" if it returns a boolean?